05week/transitions-and-transformations/css/style.css

```diff
@@ -0,0 +1,91 @@
+.grid {
+    display: flex;
```

Comment:
It would have been nice if the .grid div was a little smaller so that you got the affect of the animation when you "hovered" over the boxes